dask_mpi/tests/basic.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from time import sleep
+from distributed import Client
+from distributed.metrics import time
+from tornado import gen
+
+from dask_mpi.core import initialize
+
+scheduler_file = sys.argv[1]
+
+initialize(scheduler_file=scheduler_file)
+
+with Client(scheduler_file=scheduler_file) as c:
+    start = time()
+    while len(c.scheduler_info()['workers']) != 2:
+        assert time() < start + 10
+        sleep(0.2)
```

Comment:
Could we use a timeout in `result()` instead of this loop?

Do we want to ensure that 2 workers are started? This is not declared in this script, so this can be misleading, doesn't it?

Reply:
This test is almost exactly that the current `distributed.cli.tests.test_basic` does.  It is just a simple "port" into this package.  I agree it could be cleaned up and clarified.  It is just a first pass, in my mind.

Reply:
We aren't constructing a future here, so there is no way to use a timeout.  We're polling the scheduler to see how many workers are present.  This pattern is pretty common in our tests when there isn't an obvious thing to wait on.

Reply:
Yes sure, if we want to check that their are 2 and only 2 workers, that's fine. I was just saying we could have wait during the following instruction.

Reply:
Since this new test works (and effectively tests the new feature), I would prefer a refactor of the test(s) to be a secondary issue.  I would like to bump refactoring the test(s) to a future PR, if that's okay with everyone.

Reply:
I'm resolving this conversion to deal with test refactor at a later time.